pkgs/build-support/setup-hooks/make-binary-wrapper/make-binary-wrapper.sh

```diff
@@ -15,17 +15,17 @@ assertExecutable() {
 # makeWrapper EXECUTABLE OUT_PATH ARGS
 
 # ARGS:
-# --argv0       NAME    : set name of executed process to NAME
-#                         (otherwise it’s called …-wrapped)
+# --argv0       NAME    : set the name of the executed process to NAME
+#                         (if unset or empty, defaults to EXECUTABLE)
 # --inherit-argv0       : the executable inherits argv0 from the wrapper.
 #                         (use instead of --argv0 '$0')
-# --set         VAR VAL : add VAR with value VAL to the executable’s
-#                         environment
+# --set         VAR VAL : add VAR with value VAL to the executable's environment
 # --set-default VAR VAL : like --set, but only adds VAR if not already set in
 #                         the environment
 # --unset       VAR     : remove VAR from the environment
 # --chdir       DIR     : change working directory (use instead of --run "cd DIR")
 # --add-flags   FLAGS   : add FLAGS to invocation of executable
+# TODO(@ncfavier): --append-flags
```

Comment:
I think this will appear on `make-binary-wrapper` help. It is better to move this somewhere else.

Reply:
Where are you thinking this will appear? Are those comments parsed by anything?

Reply:
I'll remove the TODO soon™ anyway.

Reply:
> Where are you thinking this will appear? Are those comments parsed by anything?

I am almost sure it is parsed by the help itself. Let me see if I remember how to invoke it.

Reply:
What help?

Reply:
I may be misremembering, but I am almost sure that you can call `wrapProgram` manually inside a shell and it would show this text as a help if you don't pass any arguments.

This is what I am trying to remember how to do.